.eslintrc.json

```diff
@@ -0,0 +1,25 @@
+{
```

Comment:
## Your Mobile Menu
![image](https://user-images.githubusercontent.com/93039359/225311414-f1c71d9e-c279-4b0f-95d1-9ec633814c55.png)
## Figma Mobile Menu
![image](https://user-images.githubusercontent.com/41163571/225261454-e26f9f05-3c9c-4c81-8f83-942a72b4b999.png)

- [ ] It would be nice if you make your project look similar to the Figma template then it would be attractive and professional and attract the viewer's eyes, especially recruiters :+1:

**:red_circle: Question**
The previous reviewer didn't ask for changes in the design but only in functionality which i have done but now you are asking for this change in design. What's wrong with you guys? (Sorry the last part of the sentence is just an expression from a student in this situation :smile:)

**:green_circle: Answer**
The previous reviewer didn't mention this requirement because he was not able to see your mobile menu and he showed in the video that he is not able to see your mobile menu then how could he suggest changes in your design if he can't see it?
Secondly, the previous reviewer showed you the design of the expected mobile menu in his review and he said that this menu should pop up when you click on the burger menu. So it means he indirectly asked you to follow the design :+1: